lib/cli/cliCompact.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
+const CLI = require('./messages.js');
 const CLICommand = require('./cliCommand.js');
-const messages = require('./messages.js');
```

Comment:
@Apollon77 I just saw this and changed it to be in line with the other files here.

Reply:
Thank you!